src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004.ps1

```diff
@@ -0,0 +1,37 @@
+Configuration Win10_2004_L1
+{
+    Import-DSCResource -ModuleName 'CISDSC' -Name 'CIS_Microsoft_Windows_10_Enterprise_Release_2004'
+
+    node 'localhost'
+    {
+        CIS_Microsoft_Windows_10_Enterprise_Release_2004 'CIS Benchmarks'
+        {
+            #These exclusions are services that are not in default installs of Windows. Remove the exlusions if they are applicable to your envrionment.
```

Comment:
Is there anywhere in the PSDoc string thingies where we can add this so that a Get-Help will reveal this comment?

Reply:
Looking at examples on other DSC repos this looks do able. Since this would need done on all examples can we get this handled in another PR?

Reply:
Sounds good. Openend #98.